concent_api/core/tests/test_auth_report_computed_task.py

```diff
@@ -29,7 +28,7 @@ class AuthReportComputedTaskIntegrationTest(ConcentIntegrationTestCase):
     def setUp(self):
         super().setUp()
         self.compute_task_def = self._get_deserialized_compute_task_def(
-            deadline=int(dateutil.parser.parse("2017-12-01 11:00:00").timestamp())
+            deadline=self._create_timestamp_from_string("2017-12-01 11:00:00") + 1
```

Comment:
Why +1?

BTW, even if it's justified this looks like a change that should have its own commit because it does more than just "Replace manually created datetimes with helpers".

Reply:
:+1: Fixed.